spec/Body/Body.vspec

```diff
@@ -161,11 +161,53 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights
```

Comment:
Perhaps provide more description of what is a "beam" light? Does this include high and low beams?

Add a period at end of sentence.

Reply:
Thank you for your feedback. See instances of beam-lights for low and high beams.